okio-files/build.gradle

```diff
@@ -32,10 +33,11 @@ kotlin {
         implementation deps.kotlin.test.jdk
       }
     }
-    macosX64Main {
+    posixMain {
```

Comment:
Ah, I see. Yeah with no context this isn't seen as a native source set but another common source set (which you are only pulling into native source sets). I think we may need to switch to HMPP for this to work properly.